### PR TITLE
Allow the Ollama microservice to be configurable with different models

### PR DESCRIPTION
**Pull request by lvliang-intel**
## Description

Allow the Ollama microservice to be configurable with different models

## Issues

`n/a`

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

None

## Tests

Local test and pre-CI test.

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]

File: comps/llms/text-generation/ollama/llm.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 def llm_generate(input: LLMParamsDoc):
     ollama = Ollama(
```
Comment (lkk12014402) [inline comps/llms/text-generation/ollama/llm.py:22]: does the Ollama initialization have latency gap?

Comment (lvliang-intel) [reply comps/llms/text-generation/ollama/llm.py:22]: Good point, we will cover the test.

Comment (Spycsh) [reply comps/llms/text-generation/ollama/llm.py:22]: Yes, I verify and it sometimes has latency gap. Here is the default Ollama params https://github.com/ollama/ollama/blob/main/docs/modelfile.md

But I think it is safe to just use OpenAI default ones.

Comment (Spycsh) [reply comps/llms/text-generation/ollama/llm.py:22]: Also I guess this model name should be passed as input param. WIll adapt https://github.com/opea-project/GenAIComps/pull/309 later.

**Review by Spycsh — APPROVED** [review]

**Pull request merged**